main_test.go

```diff
@@ -313,3 +319,74 @@ func CreateGenericTest(t *testing.T, serverCreater func(t *testing.T), HttpMetho
 	}
 	return data, nil
 }
+
+// --------------------------------------------
+// TESTING 	PROMETHEUS INSTRUMENTATION
+// --------------------------------------------
+func TestHandlePrometheusInstrumentation(t *testing.T) {
```

Comment:
Shouldn't this test be moved in the `prometheus` dir?

Reply:
If you want, I can add separate unit tests for the objects in the prometheus dir, but this test calls the handlers defined in main, so it should stay in main_test.go.

Reply:
unit tests added